mindsdb/integrations/handlers/athena_handler/tests/test_athena_handler.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+from mindsdb.integrations.handlers.athena_handler.athena_handler import AthenaHandler
+from mindsdb.api.executor.data_types.response_type import RESPONSE_TYPE
+
+
+class AthenaHandlerTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):

```

Comment:
Can we change these tests and use mocks? As a reference, you can check https://github.com/mindsdb/mindsdb/blob/main/tests/unit/handlers/test_mysql.py#L24